.github/ISSUE_TEMPLATE/questions_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Questions and/or Comment
+about: Have a questions that needs to be answered?
```

Comment:
We might need to inform the user that general purpose questions can also be answered on our forums and discord community. So they could first try there.

Especially questions such as "how to create XXX?" or "how do I use YYY?"

Reply:
@Kryptos-FR Let me know if my latest commit works for you as a solutions and if any rewording